wordpress-mobile/WordPress-iOS
# Fixes EMBEDDED_CONTENT_CONTAINS_SWIFT warnings with CocoaPods

Fixes #5160 

To test:
1. Install CocoaPods. No warnings about `EMBEDDED_CONTENT_CONTAINS_SWIFT` should emit.
2. Build the project - no build errors.
3. Make sure `EMBEDDED_CONTENT_CONTAINS_SWIFT` is set to NO in WordPressShareExtension and WordPressTodayWidget.

Needs review: @sendhil 

Will probably cherry pick these commits into develop after this is merged.
